Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj

```diff
@@ -97,7 +97,7 @@
     <Compile Include="Tiled\TmxTerrain.cs" />
     <Compile Include="Tiled\TmxTile.cs" />
     <Compile Include="Tiled\TmxTileOffset.cs" />
-    <Compile Include="Tiled\TmxTileset.cs" />
+    <Compile Include="Tiled\TmxTileSet.cs" />
```

Comment:
Ah, I see the problem. The [filename doesn't match the class name](https://github.com/craftworkgames/MonoGame.Extended/blob/master/Source/MonoGame.Extended.Content.Pipeline/Tiled/TmxTileSet.cs).